src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeReportToFile(report, toFileName);
+    }
+
+    public String readFile(String fileName) {
```

Comment:
Methods that use only in class should be private
```suggestion
    private String readFile(String fileName) {
```